config.lua

```diff
@@ -511,84 +524,10 @@ Config.BigBanks = {
     }
 }
 
-
 Config.MaleNoHandshoes = {
-    [0] = true,
-    [1] = true,
-    [2] = true,
-    [3] = true,
-    [4] = true,
-    [5] = true,
-    [6] = true,
-    [7] = true,
-    [8] = true,
-    [9] = true,
-    [10] = true,
-    [11] = true,
-    [12] = true,
-    [13] = true,
-    [14] = true,
-    [15] = true,
-    [18] = true,
-    [26] = true,
-    [52] = true,
-    [53] = true,
-    [54] = true,
-    [55] = true,
-    [56] = true,
-    [57] = true,
-    [58] = true,
-    [59] = true,
-    [60] = true,
-    [61] = true,
-    [62] = true,
-    [112] = true,
-    [113] = true,
-    [114] = true,
-    [118] = true,
-    [125] = true,
-    [132] = true
+    [0] = true, [1] = true, [2] = true, [3] = true, [4] = true, [5] = true, [6] = true, [7] = true, [8] = true, [9] = true, [10] = true, [11] = true, [12] = true, [13] = true, [14] = true, [15] = true, [18] = true, [26] = true, [52] = true, [53] = true, [54] = true, [55] = true, [56] = true, [57] = true, [58] = true, [59] = true, [60] = true, [61] = true, [62] = true, [112] = true, [113] = true, [114] = true, [118] = true, [125] = true, [132] = true
 }
 
 Config.FemaleNoHandshoes = {
-    [0] = true,
-    [1] = true,
-    [2] = true,
-    [3] = true,
-    [4] = true,
-    [5] = true,
-    [6] = true,
-    [7] = true,
-    [8] = true,
-    [9] = true,
-    [10] = true,
-    [11] = true,
-    [12] = true,
-    [13] = true,
-    [14] = true,
-    [15] = true,
-    [19] = true,
-    [59] = true,
-    [60] = true,
-    [61] = true,
-    [62] = true,
-    [63] = true,
-    [64] = true,
-    [65] = true,
-    [66] = true,
-    [67] = true,
-    [68] = true,
-    [69] = true,
-    [70] = true,
-    [71] = true,
-    [129] = true,
-    [130] = true,
-    [131] = true,
-    [135] = true,
-    [142] = true,
-    [149] = true,
-    [153] = true,
-    [157] = true,
-    [161] = true,
-    [165] = true
+    [0] = true, [1] = true, [2] = true, [3] = true, [4] = true, [5] = true, [6] = true, [7] = true, [8] = true, [9] = true, [10] = true, [11] = true, [12] = true, [13] = true, [14] = true, [15] = true, [19] = true, [59] = true, [60] = true, [61] = true, [62] = true, [63] = true, [64] = true, [65] = true, [66] = true, [67] = true, [68] = true, [69] = true, [70] = true, [71] = true, [129] = true, [130] = true, [131] = true, [135] = true, [142] = true, [149] = true, [153] = true, [157] = true, [161] = true, [165] = true
```

Comment:
```suggestion
    [0] = true,
    [1] = true,
    [2] = true,
    [3] = true,
    [4] = true,
    [5] = true,
    [6] = true, 
    [7] = true,
    [8] = true,
    [9] = true,
    [10] = true,
    [11] = true,
    [12] = true,
    [13] = true,
    [14] = true,
    [15] = true,
    [19] = true,
    [59] = true, 
    [60] = true,
    [61] = true,
    [62] = true,
    [63] = true,
    [64] = true,
    [65] = true,
    [66] = true, 
    [67] = true,
    [68] = true,
    [69] = true,
    [70] = true,
    [71] = true,
    [129] = true,
    [130] = true,
    [131] = true,
    [135] = true,
    [142] = true,
    [149] = true,
    [153] = true,
    [157] = true,
    [161] = true,
    [165] = true
```
This format shouldn't of been changed